test/language/module-code/abstract-module-source/length.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2024 Chengzhong Wu. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-properties-of-the-%abstractmodulesource%25-intrinsic-object
+description: >
+  %AbstractModuleSource%.length property descriptor
+info: |
+  28.1.1.1 %AbstractModuleSource% ( )
+
+  The "length" property of this function is +0𝔽.
+
+includes: [propertyHelper.js]
+features: [source-phase-imports]
+flags: [module]
+---*/
+
+import source moduleSource from './name.js';
```

Comment:
This line throws a linking error, because JS modules do not have a source. We might ask hosts to expose `AbstractModuleSource` as part of the `$262` harness (`$262.AbstractModuleSource`). There is no other way within ECMA-262 to reach it.

Reply:
Thanks for the suggestion. Updated!